genrules/BUILD

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 EngFlow Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# NOTE
+# The following genrules allow for an intensive test on the building system
+# as they create thousands of targets to be specified in few lines of code.
+# WARNING: Be careful with the number of targets created as it may be
+# to much for the current free-tier configuration. 
+# Maximum of targets tested so far: 30000 (pdf file)
+#
+# Change the upper bound of the range and enjoy! 
+#
+
+# Text files with size 4KB
+[genrule(
+ name = "foo_text" + str(x),
+ outs = ["foo_text" + str(x) + ".txt"],
+ cmd = "echo 'Hello engflow world' > $@",
+) for x in range(1,101)]
```

Comment:
Make the upper bound a variable for each rule: https://docs.bazel.build/versions/main/skylark/tutorial-sharing-variables.html

You can put it at the top of the file to make it more visible

Reply:
I did the change, one shared variable for all the rules. Is it correct?